### PR TITLE
More robust deactivation

### PR DESCRIPTION
**Pull request by rubenzorrilla**
Now those elements which fixed and inactive nodes sumamation is equal (or even larger) than the total number of nodes are deactivated as well.

### CODE REVIEW

File: applications/FluidDynamicsApplication/custom_processes/distance_modification_process.h [hunk 1]
```diff
@@ -346,10 +347,11 @@ class DistanceModificationProcess : public Process
             {
                 if (rGeometry[itNode].GetSolutionStepValue(DISTANCE)<0.0)
                     inside++;
+                if (rGeometry[itNode].IsFixed(VELOCITY))
```
Comment (RiccardoRossi) [inline applications/FluidDynamicsApplication/custom_processes/distance_modification_process.h]: this makes no sense (and i guess it does not compile)

you need to check for the components

       if (rGeometry[itNode].IsFixed(VELOCITY_X))

**Review by RiccardoRossi — CHANGES_REQUESTED** [review]
you need to check for the components. The use of IsFixed you make is not correct

File: applications/FluidDynamicsApplication/custom_processes/distance_modification_process.h (hunk 1 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/custom_processes/distance_modification_process.h]: You're right. Probably I committed a wrong file version. My bad. Thanks @RiccardoRossi .

**Review by RiccardoRossi — APPROVED** [review]
+1

**Pull request merged**